bim2sim/elements/base_elements.py

```diff
@@ -132,112 +132,17 @@ def get_pending_attribute_decisions(
         all attributes functions are used to calculate the remaining attributes
         """
 
-        decisions = DecisionBunch()
-        dependant = {}
+        all_attr_decisions = DecisionBunch()
         for inst in elements:
             bunch = inst.attributes.get_decisions()
-            _decisions, _dependant = cls.extract_bunch_components(bunch, inst)
-            decisions.extend(_decisions)
-            dependant.update(_dependant)
+            attr_decisions = []
```

Comment:
I think this code section starting at line 135 can simplified to:
```suggestion
all_attr_decisions = DecisionBunch()
for inst in elements:
    bunch = inst.attributes.get_decisions()
    all_attr_decisions.extend(bunch)
```